habitat/plan.sh

```diff
@@ -112,15 +112,24 @@ do_install() {
     ohai
     test-kitchen
   )
-  for gem in "${gems_to_appbundle[@]}"; do
-    build_line "AppBundling ${gem}"
-    bundle exec appbundler $HAB_CACHE_SRC_PATH/$pkg_dirname $pkg_prefix/ruby-bin $gem
-  done
 
-  # Link the appbundled binstubs into the package's bin directory
-  mkdir -p $pkg_prefix/bin
-  for exe in $pkg_prefix/ruby-bin/*; do
-    wrap_ruby_bin $(basename ${exe})
+  build_line "Installing generated gem. (${CACHE_PATH}/${pkg_name}-${pkg_version}.gem)"
+  gem install --no-doc "${CACHE_PATH}/${pkg_name}-${pkg_version}.gem"
```

Comment:
this `gem install` command was inside the `do_build` callback, were as now it is inside the `do_install` since we are installing it 😉